camel-support/src/test/resources/application.yaml

```diff
@@ -1,12 +1,9 @@
 repository:
   step:
     jar:
-      - url: "https://repo1.maven.org/maven2/org/apache/camel/kamelets/camel-kamelets/3.21.0/camel-kamelets-3.21.0.jar"
-      - url: "resource://camel-3.19.0.zip"
-    git:
-      -
-        url: "https://github.com/KaotoIO/camel-component-metadata.git"
-        tag: "main"
+      - url: "resource://camel-kamelets-3.21.0.jar"
+      - url: "resource://camel-connectors-3.21.0.zip"
+      - url: "resource://camel-component-metadata.zip"
   viewdefinition:
     jar:
-      - url: "https://github.com/KaotoIO/kaoto-viewdefinition-catalog/archive/refs/tags/test.zip"
```

Comment:
Are we testing the remote url retrieval of resources somewhere else?

Reply:
I found it only for Kamelet, https://github.com/KaotoIO/kaoto-backend/blob/main/camel-support/src/test/java/io/kaoto/backend/camel/metadata/parser/step/kamelet/KameletParseCatalogTest.java#L71C33-L71C33

I can add something similar for each type of resource.
Or we can add another profile which will test `api` package with remote types of resources.

Reply:
I am working on that tests

Reply:
@Delawen Added tests for that https://github.com/KaotoIO/kaoto-backend/pull/818/commits/328b72fc4accbd1f8c483a1440083193a7703d2c